duck_router/lib/src/configuration.dart

```diff
@@ -80,6 +84,8 @@ class DuckRouterConfiguration {
       return;
     }
 
+    _locationFactories[location.runtimeType.toString()] = location.fromJson;
```

Comment:
Only called by DuckInformationParser._maybeIntercept or manually, so this wont execute before routes are added, or this method manually called.